src/components/Form/Form.stories.tsx

```diff
@@ -426,7 +417,7 @@ export const Playground = (): ReactElement => {
 			description: text("Description for section 1", "Description for section 1"),
 			fields: [
 				// row 1
-				[["textField"], ["textArea"], ["check"]],
+				[["textField"], ["textField"], ["check"]],

```

Comment:
I think it might be better to just delete it because then when the user types on any of them the value will apply to both and it might look as if there's an issue
```suggestion
				[["textField"], ["check"]],

```